src/service/resources-impl.js

```diff
@@ -859,6 +871,19 @@ export class Resources {
     this.visibilityStateMachine_.setState(
       this.viewer_.getVisibilityState()
     );
+
+    if (this.maybeChangeHeight_) {
```

Comment:
I'm thinking about assigning `true` to `maybeChangeHeight_` and call `doPass_` directly in the test, is it fine?